tests/main.spec.ts

```diff
@@ -1,5 +1,6 @@
 import * as main from "../src/main";
 import * as assert from "assert";
+import { describe, it } from "mocha";
```

Comment:
I prefer to not use suite and test instead of describe and it. So it does not need to import.